process/baseProcessor.go

```diff
@@ -289,6 +298,14 @@ func (bp *BaseProcessor) CallPostRestEndPoint(
 	return responseStatusCode, errors.New(genericApiResponse.Error)
 }
 
+func (bp *BaseProcessor) triggerNodesSyncCheck(address string) {
+	log.Info("triggering nodes state checks because of an offline node", "address of offline node", address)
+	select {
```

Comment:
Nice idea. Could implement later if a new use-case is desired. 
Right now, the reason is:
- every minute, the nodes' statuses are checked
- if anytime during that minute proxy detects an offline/unresponsive node, it will trigger the nodes check earlier